akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -104,18 +104,15 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
 
   @inline
   final def withMdc(logSource: String, logEvent: LogEvent)(logStatement: => Unit): Unit = {
-    MDC.put(mdcAkkaSourceAttributeName, logSource)
-    MDC.put(mdcThreadAttributeName, logEvent.thread.getName)
-    MDC.put(mdcAkkaTimestamp, formatTimestamp(logEvent.timestamp))
-    MDC.put(mdcActorSystemAttributeName, actorSystemName)
-    logEvent.mdc.foreach { case (k, v) => MDC.put(k, String.valueOf(v)) }
+    val mdcAdapter = MDC.getMDCAdapter
+    mdcAdapter.put(mdcAkkaSourceAttributeName, logSource)
```

Comment:
avoid accessing the ThreadLocal for each value

Reply:
Feels like a dejavu but if I look at hte Logback impl it accesses the thread local for each `put` as well, so there's no difference?

Reply:
you are right, misunderstand of me. I thought the MDCAdapter was *the* ThreadLocal value

an alternative would be `setContextMap`, but that creates a new copy-Map so might not be better